app/views/spree/checkout/_delivery.html.erb

```diff
@@ -61,7 +59,7 @@
           <% end %>
         </div>
 
-        <%= render '/spree/shared/ddu_ddp' if @flow_exp.try(:key) == 'canada' %>
+        <%= render '/spree/shared/ddu_ddp' if Flow::Experience.delivered_duty?(@flow_exp) %>
```

Comment:
Method name `delivered_duty?` seems unclear - If I'm reading correctly, we want to know if we are offering `customer_choice_of_duty`